tests/ml_test.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
 
 import math
+import time
```

Comment:
add time to the `test-requirements.txt` file

Reply:
done